### PR TITLE
Add initial github actions

### PR DESCRIPTION
**Pull request by jdolitsky**
Currently failing on completion tests - is this expected @marckhouzam? We can start with this and improve from here. Would rather failing CI than no CI :) 

With GitHub Actions we are able to upload the Robot test report as a .zip - check out the results here (top right under "Artifacts"): https://github.com/helm/acceptance-testing/commit/b5627d491d534cb6a63002b32c0cc89770e8f76a/checks?check_suite_id=263356948

I's also like to soon address https://github.com/helm/acceptance-testing/issues/2 (trigger CI run on helm/helm push).

### CODE REVIEW

File: scripts/github-actions-ci.sh [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+GOPATH=/tmp/gopath make build
+popd
+popd
+mv /tmp/gopath/src/helm.sh/helm/bin/helm bin/helm
+helm version
+which helm
+
+# These tools appear to be in the GitHub "ubuntu-latest" environment, but not in
+# the ubuntu:latest image from Docker Hub
+if ! [[ -x "$(command -v curl)" || -x "$(command -v pip3)" || -x "$(command -v docker)" ]]; then
```
Comment (marckhouzam) [inline scripts/github-actions-ci.sh:40]: We now also need to install git for the clone to work.
Of course, the cloning code should be moved down below the installation of git.

File: scripts/github-actions-ci.sh [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+rm -rf bin/
+mkdir -p bin/
+export PATH="${PWD}/bin:${HOME}/.local/bin:${PATH}"
+export GITHUB_SHA="${GITHUB_SHA:-latest}"
+
+# Build helm from dev-v3 source
+which helm || true
+mkdir -p /tmp/gopath/src/helm.sh
+pushd /tmp/gopath/src/helm.sh
+git clone https://github.com/helm/helm.git -b dev-v3
```
Comment (marckhouzam) [inline scripts/github-actions-ci.sh:29]: I think you can speed up the clone by adding the flags: `--single-branch --depth=1`

File: .github/workflows/acceptance-tests.yml [hunk 3]
```diff
@@ -0,0 +1,22 @@
+name: acceptance-tests
+
+on:
+  push:
+    branches:
+      - master
```
Comment (marckhouzam) [inline .github/workflows/acceptance-tests.yml:6]: Is it possible to have multiple triggers for an action?  I believe the two actions are identical except for the trigger.

**Review by marckhouzam — APPROVED** [review]
Nice!

**Comment by marckhouzam** [comment]
The completion tests still fail but for a bigger problem.  I haven't looked into it but once I have a moment I will.  The errors are:
```
/home/runner/work/acceptance-testing/acceptance-testing/acceptance-testing-reports/e8c50c5941f59d9db432bbe86986e4cff8e637a7/helm-acceptance-completion.wszNaG/completionTests.sh: line 25: 
/home/runner/work/acceptance-testing/acceptance-testing/acceptance-testing-reports/e8c50c5941f59d9db432bbe86986e4cff8e637a7/helm-acceptance-completion.wszNaG/bin/helm: No such file or directory
```

File: .github/workflows/acceptance-tests.yml (hunk 3 above)
Comment (jdolitsky) [reply .github/workflows/acceptance-tests.yml:6]: Yes, but this affects the status of the overall build. So for example, if a PR fails the "badge" on the readme would turn red. But I agree, there should be a better way to do this as to not duplicate code

**Comment by jdolitsky** [comment]
@marckhouzam going to go ahead with the merge despite the failures, lets iterate on this from here

**Pull request merged**